### ISSUE TITLE
404 accessing repo with trailing slash and nginx reverse proxy

### ISSUE THREAD

**Issue by vidkun** [gitbucket/gitbucket#1840]
Using the nginx config provided in the Wiki article for a reverse proxy, I'm getting 404 errors when accessing a repo and using a trailing slash on the url. This is currently breaking my jenkins integration.

Browsing to:
`https://gitbucket.tld/Group/Repo`
works just fine as expected.

Browsing to:
`https://gitbucket.tld/Group/Repo/`
returns a 404 from nginx.

Given the absence of logs from gitbucket during this, I'm assuming it's an nginx config issue. However, I'm using the config provided in the wiki write up for it.

This is on the latest version of Gitbucket.

**Comment by vidkun** [comment]
The error page just says:
```
HTTP ERROR 404
Problem accessing /Group/Repo/. Reason:

    Not Found
```

**Comment by takezoe** [comment]
Fixed in #1921
